Source/RageCommunity.Library/Extensions/PedExtensions.cs

```diff
@@ -27,7 +28,7 @@ public static class PedExtensions
         /// <remarks>
         /// A list of scenarios can be found <a href="https://github.com/DurtyFree/gta-v-data-dumps/blob/master/scenariosCompact.json">here</a>.
         /// </remarks>
-        public static void StartScenarioInPlace(this Ped ped, string scenarioName, int unkDelay = 0, bool playEnterAnimation = true) => NativeWrappers.TaskStartScenarioInPlace(ped, scenarioName, unkDelay, playEnterAnimation);
+        public static void StartScenarioInPlace(this Ped ped, Scenarios scenarioName, int unkDelay = 0, bool playEnterAnimation = true) => NativeWrappers.TaskStartScenarioInPlace(ped, scenarioName, unkDelay, playEnterAnimation);
```

Comment:
Why is the parameter called scenarionName when the type was changed from string to Scenarios?

Reply:
Because I changed the type and not the parameter name.